app/android/AndroidManifest.xml

```diff
@@ -1,32 +1,72 @@
-<?xml version='1.0' encoding='utf-8'?>
-<manifest package="uk.co.lutraconsulting"
-          xmlns:android="http://schemas.android.com/apk/res/android"
-          android:versionName="__BASH_REPLACE_VERSION__"
-          android:versionCode="__BASH_REPLACE_CODE__"
-          android:installLocation="auto">
-    <application android:hardwareAccelerated="true"
-                 android:name="org.qtproject.qt.android.bindings.QtApplication"
-                 android:label="Mergin Maps"
-                 android:icon="@mipmap/ic_merginmaps_launcher"
-                 android:extractNativeLibs="true"
-                 android:allowNativeHeapPointerTagging="false"
-                 android:preserveLegacyExternalStorage="true">
-        <activity android:configChanges="orientation|uiMode|screenLayout|screenSize|smallestScreenSize|layoutDirection|locale|fontScale|keyboard|keyboardHidden|navigation|mcc|mnc|density"
-                  android:name="uk.co.lutraconsulting.InputActivity"
-                  android:label="Mergin Maps"
-                  android:screenOrientation="unspecified"
-                  android:launchMode="singleTop"
-                  android:theme="@style/SplashScreenTheme">
+<?xml version="1.0"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="uk.co.lutraconsulting"
+    android:installLocation="auto"
+    android:versionCode="-- %%INSERT_VERSION_CODE%% --"
+    android:versionName="-- %%INSERT_VERSION_NAME%% --">
```

Comment:
how about the __BASH_REPLACE_VERSION__ and __BASH_REPLACE_CODE__ variables used here before?